test/output-tests/syn_functions.expected

```diff
@@ -81,12 +81,22 @@ syn_functions.at-198-prog.cob:16.11-16.15:
   18              .
 >> Error: Invalid syntax
 
-syn_functions.at-198-prog.cob:16.27-16.29:
+syn_functions.at-198-prog.cob:16.24:
   13          01  ret PIC X.
   14              
   15          PROCEDURE DIVISION RETURNING ret.
   16 >            MOVE FUNCTION x TO ret
-----                              ^^
+----                           ^
+  17              MOVE FUNCTION x TO ret
+  18              .
+>> Hint: Missing <word>
```

Comment:
There's something fishy here.

Reply:
That might be because we are in a function definition, when I change `FUNCTION-ID` and `END FUNCTION` by `PROGRAM-ID` and `END PROGRAM` in the source file, the syntax errors are no more present

Reply:
Yes we can ignore those errors: the current grammar only accepts more programs after having parsed a first program (*i.e* not a function definition, a class, etc).